asciidoctorj-distribution/src/test/groovy/org/asciidoctor/diagram/WhenDitaaDiagramIsRendered.groovy

```diff
@@ -58,7 +59,10 @@ Hello World
 
 
         then:
-        result.contains("""src="${imagesOutDir.absolutePath}/${imageFileName}.png""")
+        if (Platform.getNativePlatform().OS == Platform.OS.WINDOWS)
+            result.contains("""src="${imagesOutDir.absolutePath.replaceAll("\\\\", "//")}/${imageFileName}.png""")
+        else
+            result.contains("""src="${imagesOutDir.absolutePath}/${imageFileName}.png""")
```

Comment:
At some point the usage of 'imagesdir 'was added. We could completely remove it or just set any value for the sake of ensuring imagesdir works as espected.